src/spots/spots-views.controller.ts

```diff
@@ -128,13 +128,11 @@ export class SpotsViewsController {
 	async renderConfigureSpot(
 		@AuthUser() user: User,
 		@Param('id') id: string,
-	): Promise<Record<string, string | undefined | Spot | boolean>> {
+	): Promise<Record<string, boolean | string | Spot>> {
 		const spot = await this.spotsService.getSpot(id, user);
 
 		return {
-			username: user?.username,
-			email: user?.email,
-			role: user?.role,
+			title: 'Configure the spot | Spot in',
 			spot,
 			toConfigure: true,
```

Comment:
I think this can be replaced just by checking the user's role (`GUEST`) in the view.